tests/config.py

```diff
@@ -18,7 +15,7 @@ async def test_config_static():
     """Tests static (one-time) configuration parsing in blocking and async mode"""
 
     @asfquart.config.static
-    def config_callback(yml: dict):
+    async def config_callback(yml: dict):
```

Comment:
there was no async definition but the callback was awaited. I assume that should be an async testcase.

Reply:
yes and no. the config callback can be both sync or async, depending on what the author's intention is. the decorator tests for whether the config callback is a blocking or async function and ..does the right thing. so I do think it is intentional that the config_callback is defined as blocking here, even though the function is awaited.

Reply:
reverted and documented